deadseeker/deadseeker.py

```diff
@@ -0,0 +1,138 @@
+import asyncio
+import aiohttp
+from .linkacceptor import LinkAcceptor, LinkAcceptorBuilder
+from urllib.parse import urlparse, urljoin
+from .linkparser import LinkParser
+from aiohttp_retry import RetryClient, ExponentialRetry  # type: ignore
+from typing import List, Set, Deque
+import time
+import logging
+
+DEFAULT_WEB_AGENT: str = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)' +\
+    ' AppleWebKit/537.36 (KHTML, like Gecko)' +\
+    ' Chrome/60.0.3112.113 Safari/537.36'
+DEFAULT_RETRY_MAX_TRIES: int = 4
+DEFAULT_RETRY_MAX_TIME: int = 30
```

Comment:
The Timeout time for http requests is also no longer used right? Would love to see this being implemented, but if there is no option just remove this everywhere.

Reply:
Wow, fantastic catch, I completely missed that I wasn't using this value.   There is definitely a way to incorporate the value into the aiohttp-retry library, so I will add it

Reply:
@ScholliYT, ready for review!   This code is used in clientsessionsfactory.py